src/main/java/classify/user/Ui.java

```diff
@@ -291,4 +299,25 @@ public static void printTotalClassesAttended(int classes) {
     public static void println (String text) {
         System.out.println(text);
     }
+
+    public static void printAddSubject() {
+        System.out.println(ADD_SUBJECT);
+    }
+
+    public static void printInvalidResponse() {
+        System.out.println(INVALID_RESPONSE);
+    }
```

Comment:
Good use of magic literals and implemented in such a way so that the messages can be reused easily! LGTM! KUTGW!